features/login.feature

```diff
@@ -0,0 +1,75 @@
+Feature: Login behavior
+@QA
+Scenario: Test login logic: error handling, demo mode, and normal behavior
+    Then I install the ccz app at "test_list_search.ccz"
+
+    # normal login
+    Then I login with username "user_with_no_data" and password "123"
+    # TODO: consider counting the home screen buttons
+    Then I verify that all home buttons are present
+    Then I logout
+
+    # login in landscape mode
+    # which is way more annoying than landscape mode...
+    Then I rotate to landscape
+    Then I close the keyboard
+    Then I clear input field number 1
+    Then I close the keyboard
+    Then I clear input field number 2
+    Then I close the keyboard
+    Then I enter "user_with_no_data" into input field number 1
+    Then I close the keyboard
+    Then I enter "123" into input field number 2
+    Then I close the keyboard
+    Then I press login
+    Then I wait for progress
+
+    # TODO: consider counting the home screen buttons
+    Then I logout
+    Then I rotate to portrait
+    Then I wait
+
+    # login in demo mode
+    Then I select "Enter as Demo User" menu item
```

Comment:
Hey so this step is now failing for me (I couldn't get to this point before because of the failure above). The reason is that the call to `Then I logout` on line 28 doesn't actually work, seemingly because in landscape mode, the logout button is not visible, and I don't think the way `Then I logout` is implemented provisions for that. Are you not experiencing the same issue when you run it? Maybe your phone shows more in landscape mode than mine does, but I think the way we look for home screen buttons to press needs to be more reobust